File: teaLeaf-ruby-Challenges-series/fizzbuzz.rb
class FizzBuzz
  attr_accessor :arry

  def initialize(max_num)
    @arry = [*1..max_num]
  end

  def fizz_buzz
    arry.each do |num|
      print num.to_s if num % 3 != 0 and num % 5 !=0
      print "#{"Fizz" if num % 3 == 0}" + "#{"Buzz" if num % 5 == 0}"
      print " "
    end
  end

end

FizzBuzz.new(100).fizz_buzz

File: 301/javascript/fizz_buzz.js
function fizz_buzz() {
  for (var i = 1; i < 101; 1++) {
    output = "";
    if (i / 3 == 0) {
      output = "Fizz";
    }

    if (i / 5 == 0) {
      output = output + "Buzz";
    } else {
      output = i;
    }
    console.log(output);
  }
}